### PR TITLE
Add social media images to Ask categories

### PR DESCRIPTION
**Pull request by contolini**
This feeds off #2952. Editors would like AskCFPB answer pages to have social media images corresponding to pages' categories. Logic has been added to pass the page's `social_sharing_image` to the template's meta tag if an image has been manually selected. Otherwise, use the page's category's image.

## Additions

- Image chooser to both Answer and Category models

## Testing

```
./drop-db.sh
./create-mysql-db.sh
mysql -uroot v1 < ~/Downloads/production_django.sql
python cfgov/manage.py migrate
python cfgov/manage.py runscript migrate_knowledgebase
```

1. Edit a category and add an image via the image chooser panel.
1. Load a live answer page that belongs to the above category.
1. View the page's source and look for `<meta property="og:image" content="*THE IMAGE YOU SELECTED*">`.

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/ask_cfpb/tests/models/test_pages.py [hunk 1]
```diff
@@ -72,6 +73,10 @@ def setUp(self):
             SubCategory, name='stub_subcat', parent=self.category, _quantity=3)
         self.category.subcategories.add(self.subcategories[0])
         self.category.save()
+        self.social_image = mommy.prepare(CFGOVImage)
+        self.social_image.save()
```
Comment (chosak) [inline cfgov/ask_cfpb/tests/models/test_pages.py]: You can use [`mommy.make`](http://model-mommy.readthedocs.io/en/latest/basic_usage.html#basic-usage) to prepare and save the object in one step.

File: cfgov/ask_cfpb/tests/models/test_pages.py [hunk 2]
```diff
@@ -737,3 +742,37 @@ def test_get_nonexistent_reusable_text_snippet(self):
...
+        category_page = self.create_category_page(ask_category=self.category)
+        self.assertIsNone(category_page.meta_image)
+
+    def test_category_meta_image(self):
+        category = mommy.make(Category, category_image=self.social_image)
+        category_page = self.create_category_page(ask_category=category)
+        self.assertEqual(category_page.meta_image, self.social_image)
+
+    def test_answer_meta_image_empty(self):
+        self.assertIsNone(self.page2.meta_image)
```
Comment (chosak) [inline cfgov/ask_cfpb/tests/models/test_pages.py]: This kind of test is a little bit hard to follow as a reader would have to figure out what `self.page2` is supposed to be.

File: cfgov/ask_cfpb/tests/models/test_pages.py (hunk 1 above)
Comment (contolini) [inline cfgov/ask_cfpb/tests/models/test_pages.py]: 👍
